packages/components/pie-modal/package.json

```diff
@@ -43,6 +38,12 @@
     "extends": "../../../package.json"
   },
   "dependencies": {
+    "@justeat/pie-design-tokens": "5.8.2",
+    "@justeattakeaway/pie-button": "0.36.0",
+    "@justeattakeaway/pie-icon-button": "0.21.0",
+    "@justeattakeaway/pie-icons-webc": "0.11.1",
+    "@justeattakeaway/pie-spinner": "0.1.0",
+    "@justeattakeaway/pie-webc-core": "0.11.0",
```

Comment:
Why are we moving these? 🤔

Reply:
I don't believe having them as a `devDependency` is the correct thing here. these dependencies are needed for the component to function and have to be shipped in the production bundle and, are not only needed for development purposes such as testing or storybook-related stuff.